src/components/AppRoot/AppRoot.tsx

```diff
@@ -52,13 +57,14 @@ export const AppRoot = withAdaptivity<AppRootProps>(
     hasMouse,
     noLegacyClasses = false,
     scroll = "global",
+    portalRoot,
     ...props
   }) => {
     // normalize mode
     const mode = _mode || (_embedded ? "embedded" : "full");
     const isKeyboardInputActive = useKeyboardInputTracker();
     const rootRef = React.useRef<HTMLDivElement | null>(null);
-    const [portalRoot, setPortalRoot] = React.useState<HTMLDivElement | null>(
+    const [_portalRoot, setPortalRoot] = React.useState<HTMLDivElement | null>(
       null
     );
```

Comment:
указать 

```
    portalRoot: portalRootProp = null
```
и потом
```suggestion
    const [_portalRoot, setPortalRoot] = React.useState<HTMLDivElement | null>(portalProp);
```
мб так? и тогда в 96 строчке вообще выходить из эффекта если `portal` есть

Reply:
Ага, так лучше) Поправил